api/src/services/graphql/index.ts

```diff
@@ -1055,6 +1046,19 @@ export class GraphQLService {
 						},
 					});
 				}
+
+				const eventName = `${collection.collection}_mutated`.toUpperCase();
+				const subscriptionName = camelCase(eventName);
+				const subscriptionType = ReadCollectionTypes[collection.collection].clone(collection.collection + '_mutation');
+				subscriptionType.addFields({
+					event: GraphQLString,
+				});
+				schemaComposer.Subscription.addFields({
+					[subscriptionName]: {
+						type: subscriptionType,
+						subscribe: createSubscriptionGenerator(self, eventName, subscriptionName),
+					},
+				} as any);
```

Comment:
Seems like because the for loop wrapping this subscription logic has this line at the top:

https://github.com/directus/directus/blob/d23f2c0c1a538c30203d26e14208c962d02c9bc5/api/src/services/graphql/index.ts#L803

`directus_collections`, `directus_fields`, and `directus_relations` are currently not available via GraphQL Subscriptions:

https://github.com/directus/directus/blob/d23f2c0c1a538c30203d26e14208c962d02c9bc5/api/src/services/graphql/index.ts#L94-L97

Reply:
You cannot query these with GraphQL either. This is a limitation of the current GraphQL implementation and i consider this out of scope for this PR.